examples/i18n/src/lib.rs

```diff
@@ -0,0 +1,179 @@
+// Some Clippy linter rules are ignored for the sake of simplicity.
+#![allow(clippy::needless_pass_by_value, clippy::trivially_copy_pass_by_ref)]
```

Comment:
This line was just copied over from counter example. Deleted. I ran "cargo make verify" directly after cloning seed repo. It fails with 21 errors mostly related to thread safety.

Not sure what is going on. Maybe there is something wrong in my dev environment?

`
[cargo-make] INFO - Execute Command: "cargo" "clippy" "--all-features" "--" "--deny" "warnings" "--deny" "clippy::pedantic" "--deny" "clippy::nursery"
    Checking seed v0.7.0 (/home/tobias/IdeaProjects/seed)
error: future cannot be sent between threads safely
  --> src/app/cmds.rs:26:6
   |
26 | ) -> impl Future<Output = MsU> {
   |      ^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `-D clippy::future-not-send` implied by `-D clippy::nursery`
   = note: `impl FnOnce() -> MsU + Clone + 'static` doesn't implement `std::marker::Send`
   = note: `*mut u8` doesn't implement `std::marker::Send`
   = note: `(dyn std::ops::FnMut() + 'static)` doesn't implement `std::marker::Send`
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#future_not_send

error: this could be a `const fn`
   --> src/browser/fetch/request.rs:159:5
    |
159 | /     pub fn timeout(mut self, timeout: u32) -> Self {
160 | |         self.timeout = Some(timeout);
161 | |         self
162 | |     }
    | |_____^
    |
    = note: `-D clippy::missing-const-for-fn` implied by `-D clippy::nursery`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#missing_const_for_fn

[...]

error: aborting due to 21 previous errors

error: could not compile `seed`.

To learn more, run the command again with --verbose.
[cargo-make] ERROR - Error while executing command, exit code: 101
[cargo-make] WARN - Build Failed.
`

Reply:
According to Clippy [changelog](https://github.com/rust-lang/rust-clippy/blob/master/CHANGELOG.md), the lint rule `future_not_send` is a new one and available only on Clippy beta / Rust 1.44. So I think you use nightly Rust, switch to stable and it should be ok. 
And I don't want to resolve these new Clippy problems because it may cause conflicts with the stable Clippy.

Reply:
I'm not sure if there is explicitly said in the CONTRIBUTING.md that you should switch to `stable` Rust - if it's missing I would be glad for PR.

Reply:
Ah, that's it. I'm using nightly Rust. Agree. I send a different PR for CONTRIBUTING.md.